workshops/ansible_engine/README.md

```diff
@@ -0,0 +1,19 @@
+# Ansible Workshops
```

Comment:
Previously there was no "landing page" for the engine workshops, this is added with this file.